src/main.cpp

```diff
@@ -2862,6 +2864,25 @@ uint64_t Session::GetTimeshiftBufferStart()
     return 0ULL;
 }
 
+void Session::StartReader(
+    STREAM* stream, uint64_t seekTimeCorrected, int64_t ptsDiff, bool preceeding, bool timing)
+{
+  bool bReset;
+  if (timing)
+    seekTimeCorrected += stream->stream_.GetAbsolutePTSOffset();
+  else
+    seekTimeCorrected -= ptsDiff;
+  stream->stream_.seek_time(
+      static_cast<double>(seekTimeCorrected / STREAM_TIME_BASE),
+      preceeding, bReset);
+  if (bReset)
+    stream->reader_->Reset(false);
+  bool bStarted(false);
```

Comment:
`bool bStarted = false;` is easier to understand. Especially for basic types, otherwise they can appear like constructors.